ui/00_message/pawn/p0001.xml

```diff
@@ -5,48 +5,61 @@
   </message>
   <message>
     <original>我らはいつも、覚者さまとともに</original>
+	<translation>The Rift is always with us.</translation>
   </message>
   <message>
     <original>覚者さま、お力添えの機会
 お待ちしています</original>
+	<translation>Master, I look forward to assisting you.</translation>
   </message>
   <message>
     <original>覚者さま、どうぞ私にも同伴の許可を
 リムストーンからのお声掛けを
 お待ちしています</original>
+	<translation>Please allow me to accompany you master.</translation>
```

Comment:
Ah, didn't see the whole, original message before. The second bit, from リムストーンから:
"I'm waiting for your call (from the rimstone)."
Don't know that the rimstone bit needs to be included as, though it's in the original, it sounds unnatural to mention it in English. I also don't remember the circumstances, but it could be "I've been waiting" instead of "I'm waiting" if it's a line that's played when a pawn joins. If it's a line in the home or craft instance, it's probably "I'm waiting".

Also, probably need a comma after "you" and before "master",

Reply:
I'm wondering if the "rimstone" are actually relation to "rift crystals" which are needed to summon pawns. I would make sense for a pawn to be waiting for the use of rift crystals. But I'm not sure how to word it correctly to make sense.

Reply:
Ahh its Riftstone that it is referring to